src/web/mjs/connectors/Team1x1.mjs

```diff
@@ -30,30 +49,25 @@ export default class Team1x1 extends Connector {
         return new Manga(this, id, title);
     }
 
-    async _getMangaListFromPages(mangaPageLinks, index) {
-        index = index || 0;
-        const request = new Request(mangaPageLinks[index], this.requestOptions);
-        const data = await this.fetchDOM(request, 'div#page-manga div.container div.last-post-manga div.thumb div.info h3 a', 5);
-        let mangaList = data.map(element => {
+    async _getMangas() {
+        let mangaList = [];
+        for (let page = 1, run = true; run; page++) {
+            const mangas = await this._getMangasFromPage(page);
```

Comment:
How about an alert in case getMAngasFromPage (the fetch) fails and throws an exception.
Message could be something like "The request failed, please check if the domain in the settings is still ok".